helpers/weather.js

```diff
@@ -10,17 +10,25 @@ module.exports.urlBuilder = (city) =>
  */
 module.exports.formatWeatherData = (weatherJsonData = {}) =>
 {
-    const currentDate = new Date();
-
     return {
-        city: weatherJsonData.name,
-        date: `${ currentDate.getDate() }-${ currentDate.getMonth() }-${ currentDate.getFullYear() }`,
+        city: weatherJsonData.name.toLowerCase(),
+        date: this.getToday(),
         icon: `${ process.env.WEATHER_API_IMAGE_PATH }${ weatherJsonData.weather[ 0 ].icon }${ process.env.IMAGE_SUBFIX }`,
-        temperature: weatherJsonData.main.temp + '°C',
+        temperature: weatherJsonData.main.temp,
         description: weatherJsonData.weather[ 0 ].description,
         feels_like: weatherJsonData.main.feels_like,
-        max_temperature: weatherJsonData.main.temp_max + '°C',
-        min_temperature: weatherJsonData.main.temp_min + '°C',
+        max_temperature: weatherJsonData.main.temp_max,
+        min_temperature: weatherJsonData.main.temp_min,
```

Comment:
why do we use `snake_case` instead of `camelCase` here?

Reply:
it comes from open weather API's response